addons/api/addon/serializers/worker.js

```diff
@@ -0,0 +1,51 @@
+import ApplicationSerializer from './application';
+
+export default class WorkerSerializer extends ApplicationSerializer {
+  //=methods
+
+  /**
+   * @override
+   * @method serialize
+   * @param {Snapshot} snapshot
+   */
+
+  serialize(snapshot) {
```

Comment:
Do we actually need this serialization? The tags are read only so they shouldn't be serialized I imagine.

Maybe we only need normalization? I'm not sure if by default we convert `snake_case` to `camelCase` for the models so maybe someone else can chime in

Reply:
By default we use snake case in Ember.  Mirage models, however, hydrate to camel case.  It's a little confusing because the factories use snake case.  And for read-only fields no serialisation support should be required.